test/Plutip/Contract.purs

```diff
@@ -701,6 +707,58 @@ suite = do
               "e8cb7d18e81b0be160c114c563c020dcc7bf148a1994b73912db3ea1318d488b"
           ]
 
+    test "GetDatumsByHashes" do
+      let
+        distribution :: InitialUTxOs
+        distribution =
+          [ BigInt.fromInt 5_000_000
+          , BigInt.fromInt 2_000_000_000
+          ]
+
+        datum1 :: Datum
+        datum1 = Datum $ Integer $ BigInt.fromInt 1
+
+        datum2 :: Datum
+        datum2 = Datum $ Integer $ BigInt.fromInt 2
+
+        datums :: Array Datum
+        datums = [ datum2, datum1 ]
+
+        hashes :: Maybe (Array DataHash)
+        hashes = sequence <<< map datumHash $ datums
+
+        payToTest :: ValidatorHash -> Contract () TransactionHash
+        payToTest vhash = do
+          let
+            constraints =
+              Constraints.mustPayToScript
+                vhash
+                datum1
+                Constraints.DatumWitness
+                (Value.lovelaceValueOf $ BigInt.fromInt 1_000_000)
+                <> Constraints.mustPayToScript
+                  vhash
+                  datum2
+                  Constraints.DatumWitness
+                  (Value.lovelaceValueOf $ BigInt.fromInt 1_000_000)
+
+            lookups :: Lookups.ScriptLookups PlutusData
+            lookups = mempty
+          buildBalanceSignAndSubmitTx lookups constraints
+
+      withWallets distribution \alice -> do
+        dhashes <- liftM (error "Couldn't get hashes for datums [1,2]") hashes
+        withKeyWallet alice do
+          validator <- AlwaysSucceeds.alwaysSucceedsScript
+          let vhash = validatorHash validator
+          logInfo' "Running GetDatums submittx"
+          txId <- payToTest vhash
+          awaitTxConfirmed txId
+          logInfo' "Tx submitted successfully, trying to fetch datum from ODC"
+
+          logInfo' <<< show =<< getDatumsByHashes dhashes
+          logInfo' <<< show =<< getDatumsByHashesWithErrors dhashes
```

Comment:
We can check the returned datums were correct, right? I don't think this test fails if they weren't atm